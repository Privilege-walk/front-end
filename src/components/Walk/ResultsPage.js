import React from "react";
import Typography from '@mui/material/Typography';
import Container from '@mui/material/Container';

export default function ResultsPage(){
    return (
        <Container>
            <Typography variant="h3" component="div">User Results page</Typography>
        </Container>
    );
}