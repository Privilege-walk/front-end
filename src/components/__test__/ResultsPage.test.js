import React from "react";
import { render } from '@testing-library/react';
import { act } from 'react-dom/test-utils';

import ResultsPage from '../Walk/ResultsPage';

test('renders without crashing',  () => {
    let utils;
    act (() => {
        utils = render( <ResultsPage />);
    }); 
    const { asFragment } = utils;
    expect(asFragment()).toMatchSnapshot();
})