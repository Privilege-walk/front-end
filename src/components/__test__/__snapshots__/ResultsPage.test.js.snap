// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-x9z34i-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column css-aqz4ao-MuiGrid-root"
    >
      <div
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        User Results page
      </div>
    </div>
  </div>
</DocumentFragment>
`;
